.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 14.15.5
```

Comment:
Tools like `asdf` can read this to determine which version to use when `cd`ing into the directory.